test/integration.js

```diff
@@ -46,7 +46,7 @@ describe('integration tests:', function () {
             cb(undefined, item);
           });
         vorpal
-          .command('overwrite me')
+          .command('overwrite_me')
```

Comment:
Introducing `bash-parser`, this test fail, because when you `.exec` 'overwrite me', only `overwrite` is parsed as command name. `me` is parsed as argument.
Changing the command name to `overwrite_me` make the test pass.